packages/dds/tree/src/simple-tree/api/tree.ts

```diff
@@ -424,6 +427,22 @@ export interface TreeView<TSchema extends ImplicitFieldSchema> extends IDisposab
 	readonly schema: TSchema;
 }
 
+/**
+ * A few methods from TreeView, with proposed changes to the schema aware typing to allow use with `UnsafeUnknownSchema`.
+ * @alpha
+ */
+export interface TreeViewAlpha<
```

Comment:
This is kind of a weird situation, since for my upcoming work (adding TreeContext and branching), I would _also_ need to add a TreeViewAlpha, but it isn't really the same thing as this one. In my case, I would want TreeViewAlpha to have all of the members of TreeView, _plus some more_.

Are you okay with me coopting this type for my purposes in the future? The upshot will be that TreeViewAlpha will actually have two "different" alpha APIs on it - the branching stuff and also the improved typing here. I think that's okay, because we can still promote them from alpha to beta/public on different schedules if we want. It's just a little strange.

Reply:
I avoided including all the existing members because its messy, but I know how to do it and might as well if you will need it.

Reply:
Either way (you now or me later) is fine with me.

Reply:
done.

Reply:
Thank you!